packages/edge-config/src/index.common.test.ts

```diff
@@ -216,6 +216,18 @@ describe('connectionStrings', () => {
       expect(typeof pkg.createClient).toBe('function');
     });
 
+    describe('connection', () => {
+      it('should contain the info parsed from the connection string', () => {
+        expect(edgeConfig.connection).toEqual({
+          baseUrl: 'https://example.com/ecfg-2',
```

Comment:
I haven't fully made up my mind whether we want to expose all these fields or whether we should start with only exposing the id <picture data-single-emoji=":thinkies:" title=":thinkies:"><img class="emoji" width="20" height="auto" src="https://single-emoji.vercel.app/api/emoji/eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..su-Bm6nS9vdINlZd.eoALa702jkenemzL6gomjJWBPxD2Z4y06k_6-K_e4nHCBWl-ZGPv22EKT6ZqYphv5rPWoWlgtwsG81J7U-rmj8iPMVEXgeDXiG_TG1UO8am6zJK1B9PbvSGt.uMw8rHwKGai09iB2Z86odA" alt=":thinkies:" align="absmiddle"></picture>